docs/reference/slm/apis/slm-api.asciidoc

```diff
@@ -0,0 +1,58 @@
+[role="xpack"]
+[testenv="basic"]
+[[snapshot-lifecycle-management-api]]
+== {slm-cap} API
+
+You use the following APIs to set up policies to automatically take snapshots and 
+control how long they are retained. 
+For more information about snapshot lifecycle management (SLM), see <<snapshot-lifecycle-management>>.
+
+[float]
```

Comment:
`[float]` is valid here, but you can use `[discrete]` if you'd like these headings to show up in the on-page TOC.